components/Forms/OrderForm/index.jsx

```diff
@@ -23,7 +23,7 @@ const OrderForm = () => {
             const newOrder = { customer, email, total };
             console.log(newOrder)
             const baseUrl = process.env.BASE_URL
-            await axios.post(`${baseUrl}/api/orders`, newOrder);
```

Comment:
Why have you moved away from using variable here?

Reply:
to run it first in localhost, as baseUrl variable is declared to the link of my project's deployment